domain/src/test/java/org/oppia/domain/oppialogger/exceptions/ExceptionsControllerTest.kt

```diff
@@ -293,6 +357,14 @@ class ExceptionsControllerTest {
       return application
     }
 
+    @ExperimentalCoroutinesApi
+    @Singleton
+    @Provides
+    @TestDispatcher
+    fun provideTestDispatcher(): CoroutineDispatcher {
```

Comment:
Please don’t do this. You should include the test dispatcher module, instead.

Reply:
Done.